stable/sumologic-fluentd/templates/daemonset.yaml

```diff
@@ -237,15 +243,15 @@ spec:
         {{- end }}
     {{- if .Values.tolerations }}
       tolerations:
-    {{ toYaml .Values.tolerations | indent 8 }}
+{{ toYaml .Values.tolerations | indent 8 }}
```

Comment:
Are you sure about this indentation change? Doesn't seem correct.

Reply:
Yes, because the whitespace before the template statement is prepended to the _first_ line of the templated yaml which breaks formatting.

Ergo, if the templated yaml is the following:
```
- effect: NoExecute
  operator: Exists
```

Then the resulting template is:
```
      tolerations:
                - effect: NoExecute
        operator: Exists
```
which is obviously incorrect. It was this breakage while doing routine updates that originally led me down the rabbit whole of having to update the chart for the new version of the sumologic collector.